draft-birkholz-scitt-software-use-cases.md

```diff
@@ -211,45 +211,38 @@ There is no standardized way to:
+* reliabily discern an update that has been signed by the appropriate and intended signing identity
+* clearly communicate firmware revisioning and security checks completed to ensure the firmware is safe for install
 
 ## Software Integrator assembling a software product for a smart car
 
-Software Integration is a complex activity. This typically involves getting various software components from multiple suppliers and producing an integrated package deployed as part of device assembly.
-
-Car manufacturers source integrated software for their autonomous vehicles from third parties that integrates software components from various sources. Integration complexity creates a higher risk of security vulnerabilities to the delivered software.
-
-### Software Problem Summary
+Software Integration is a complex activity. This typically involves getting various software components from multiple suppliers and producing an integrated package deployed as part of device assembly. For example, car manufacturers source integrated software for their autonomous vehicles from third parties that integrates software components from various sources. Integration complexity creates a higher risk of security vulnerabilities to the delivered software.
 
-* While the software runs on the automated vehicle, periodic vulnerability scanning software detects a known security issue with one component. End User gets a "Warning Indicator" on the dashboard. As a result it reports the problem to the car manufacturer. It is then subsequently notified to the integrator.
+Consumer of an integrated software wants:
 
-* Integrator analysis leads to a suspected issue with the supplied Operating System (OS) software from an Independent Software Vendor (ISV). It demands specific environment and architectural details associated with the built OS binary to ascertain that the software was produced without tampering by Vendor
+* Bill of material capturing all software components used, with the SBOM stored in a secure tamperproof location
+* Verifiable proofs on build process and build environment with all supplier tiers to ensure end to end build quality and security
+* Vulnerability scan reports and approved malware mitigation plan before integration
 
-* Unfortunately, there is no way for the integrator to know if the binary was compromised, so the integrator is concerned they may have delivered malware unknowingly to their customers.
-
-* ISV attempts to show that it did all the steps correctly. It does disclose information about the binary they delivered. In addition, they also reveal their build environment and the architecture they used during the build.
+There is no standardized way to:
 
-* However, there are no "Verifiable Proofs" of the statement made by ISV. All the stakeholders, in the ecosystem (end user, car manufacturer and the integrator) has to trust without any ability to verify the claims made by the ISV.
+* Provide SBOMs in a tamperproof ledger for customer consumption
```

Comment:
Suggest: Notify software integrators of vulnerabilities identified during security scans of running software.